requirements.txt

```diff
@@ -1,3 +1,3 @@
 flake8>=3.8.1
-libcst>=0.3.10
+libcst>=0.3.13
```

Comment:
@Kronuz just released 0.3.15 today which should fix the issue.
@dkteo can you revert this change?

Reply:
will do!